modules/node_modules/@ncigdc/components/MutationTable.js

```diff
@@ -118,7 +118,7 @@ const MutationTable: TMutationTable = ({
               </span>
             }
           >
-            # Affected Cases<br /> Across all Projects
+            # Affected Cases<br /> all Projects
```

Comment:
yeah I forgot the `in`. But thinking about it, `all` is not a preposition, article or conjunction in titles. It looks like a title because all the other words are capitalized.
<img width="655" alt="screen shot 2017-03-06 at 12 22 13 pm" src="https://cloud.githubusercontent.com/assets/1314446/23621158/92755104-0267-11e7-8d7d-5737937aa21a.png">
:woman_shrugging: